src/overview/fraud-events.md

```diff
@@ -13,15 +13,17 @@ Any given fraud event can be of one of the following types:
 - `StolenDevice` — any fraud that involves a stolen device; these are usually reported either by customers or law enforcement.
 - `IPFraud` — any fraud that involves a specific IP address; these are usually detected by means of behavioral analysis, geolocation checks, and overall action velocity of any given IP address.
 - `SMSA2P` — type of fraud that utilizes the rerouting of SMS through SIM boxes, disguising them as legitimate messages; these are regularly spotted manually by operators monitoring traffic patterns, contents and sending rates of any given phone number.
+- `Flash Calling` — a type of fraud that exploits missed call notifications from unknown numbers to verify users or trigger callbacks, often bypassing traditional verification methods and monitoring through traffic analysis and call patterns.
+- `Scam` — a type of fraud that deceives users through social engineering tactics, such as phishing calls, messages, or emails, often using phone numbers, email addresses, or URLs to steal sensitive information or money, typically identified by analyzing unusual transaction patterns and user reports.
 
 ## Fraud event data model
 
 Every fraud event that is submitted and stored on the blockchain adheres to the following model:
 
 | Data | Description |
 | --- | --- |
-| Fraud Identifier | The main identifier of the fraud event. <br> Could be one of the following: <ol><li>A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code.</li><li>A range of **IP addresses** (or a single one).</li><li>A unique **IMEI identifier** of a device. </li></ol> |
-| Fraud Type | The [type of the fraud event](#types-of-fraud-events). <br> Could be one of the following: <ol><li>`Wangiri`</li><li>`IRSF`</li><li>`StolenDevice`</li><li>`IPFraud`</li><li>`SMSA2P`</li></ol> |
+| Fraud Identifier | The main identifier of the fraud event. <br> Could be one of the following: <ol><li>A range of **phone numbers** (or a single one) compliant with the E.164 international standard and containing a valid National Destination Code.</li><li>A range of **IP addresses** (or a single one).</li><li>A unique **IMEI identifier** of a device. </li></li><li>A range of **E-mails** (or a single one).</li><li>A range of **URLs** (or a single one).</li></ol> |
```

Comment:
ranges of emails and urls are not possible, just a single value for the types

Reply:
Edited